pkgs/by-name/ed/edmarketconnector/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3Full,
+  makeWrapper,
+  makeDesktopItem,
+}:
+let
+  pythonEnv = python3Full.buildEnv.override {
```

Comment:
Any reason we need python3Full? If so, please document it here using a comment.

Reply:
If I remember correctly it was because of issues with `tkinter`. I'll look into it again to be sure.

Reply:
639735248e5d136fb7a59c488df63034f9f363b8